api/server/sdk/bucket_ops.go

```diff
@@ -59,7 +59,7 @@ func (s *BucketServer) Create(
 	logrus.Infof("Create bucket request received for Bucket: %s, Region: %s", name, region)
 
 	// Create bucket
-	id, err := s.driver(ctx).CreateBucket(name, region, req.AnonymousBucketAccessMode)
+	id, err := s.driver(ctx).CreateBucket(name, region, req.Endpoint, req.AnonymousBucketAccessMode)
```

Comment:
Nit......We might want to update req.Endpoint -> req.GetEndpoint() and req.AnonymousBucketAccessMode -> req.GetAnonymousBucketAccessMode().